src/test/unit/profile-tree.test.js

```diff
@@ -41,6 +45,39 @@ describe('unfiltered call tree', function() {
   }
 
   /**
+   * Before creating a ProfileTree instance some timings are pre-computed.
+   * This test ensures that these generated values are correct.
+   */
+  describe('computed counts and timings', function() {
```

Comment:
This was a new test that I wrote to assert that this code was behaving properly.